pkg/api/api.proto

```diff
@@ -294,6 +295,7 @@ message Environment {
     bool undeployVersion = 6;
     ArgoCD argoCD = 7;
     DeploymentMetaData deploymentMetaData = 8;
+    string displayVersion = 9;
```

Comment:
Shouldn't this field be part of the `Release` message?

Reply:
yes, good point